packages/wallet/src/pages/transfer/pay.tsx

```diff
@@ -0,0 +1,76 @@
+import { AppLayout } from '@/components/layouts/AppLayout'
+import { Button } from '@/ui/Button'
+import { z } from 'zod'
+import Image from 'next/image'
+import { Form, useZodForm } from '@/ui/forms/Form'
+import { Input } from '@/ui/forms/Input'
+import { Select } from '@/ui/forms/Select'
+import { Badge } from '@/components/transfer/Badge'
+import { TransferHeader } from '@/components/transfer/TransferHeader'
+
+const paySchema = z.object({
+  fromAccount: z.string(),
+  incomingPaymentUrl: z.string(),
+  amount: z.coerce.number(),
+  currency: z.string()
+})
+
+export default function Pay() {
+  const form = useZodForm({
+    schema: paySchema
+  })
+
+  const handleSubmit = form.handleSubmit((data) => {
+    console.log(data)
+  })
+
+  return (
+    <AppLayout>
+      <div className="flex flex-col lg:w-2/3">
+        <TransferHeader type="Pink" balance="$10.000" />
+        <Form form={form} onSubmit={handleSubmit}>
+          <Badge text="from" />
+          <Select
+            name="fromAccount"
+            setValue={form.setValue}
+            error={form.formState.errors.fromAccount?.message}
+            options={[]}
+            label="Account"
+          />
```

Comment:
```suggestion
        <div className="space-y-1">
          <Badge text="from" />
          <Select
            name="fromAccount"
            setValue={form.setValue}
            error={form.formState.errors.fromAccount?.message}
            options={[]}
            label="Account"
          />
        </div>
```
Ditto for every badge. I try to avoid using `top`, etc. to position an element.